index.rst

```diff
@@ -147,176 +246,121 @@ The white light source and monochromator sit on a table surrounded by sturdy bla
+      .. figure:: /_static/step_5.png
+         :name: step_5
+         :target: ../_images/step_5.png
+         :scale: 50 %
 
-You can now power everything up. Initialize the monochromator by logging into the windows machine via remote desktop.
+
+Chiller Maintenance
+-------------------
+
+You may need to refill the water in the chiller occassionally. This should be done with distilled water mixed with 10% of methanol.
+
+.. figure:: /_static/fill_cover.png
+   :name: fill_cover
+   :target: ../_images/fill_cover.png
+   :alt: fill_cover
+   :scale: 50 %
+
+   Chiller cap for refilling water/methanol mixture.
+
+Using Fiber Illuminator 
+-----------------------
+You might want to use the fiber illuminator to calibrate the fiber spectrograph. This spectral calibration lamp will be kept in storage until needed. When you want to use it, follow these steps:
+
+#. Open the right door 
+#. Plug in the fiber illuminator to power (12V). The power cable should be sitting right next to the monochromator
+#. Remove the end of the fiber from its place in the optical path and install it into the fiber illuminator 
+
+   .. note:: You may actually want to install an absorber between the lamp and the fiber spectrograph
+
+#. Turn on the illuminator and take spectra with the fiber spectrograph
+
+.. figure:: /_static/fib_lamp.png
+   :name: fib_lamp
+   :target: ../_images/fib_lamp.png
+   :alt: fib_lamp
+
+   Fiber Illuminator, which will stay in storage until needed.
+
+Monochromator Initialization
+----------------------------
+
+When the system, inlcuding the monochromator windows machine, is power cycled, you need to Initialize the monochromator before using. This can be done using Remote Desktop using the IP address listed above for the monochromator. The username is **MONOCHROMATOR-P\Monochromator**. Ask Patrick Ingraham or Parker Fagrelius for the password. 
+
+When you've logged in to the machine, open the LabView program. This will initialize the monochromator. If you have issues connecting to the hardware, you may have to check the USB connection and power cycle the monochromator itself.
 
 .. figure:: /_static/mono_screen.png
    :name: mono_screen
    :target: ../_images/mono_screen.png
    :alt: mono _screen
    :scale: 10 %
 
-If you can't talk to the LabJack, then use a direct connection via USB to the LabJack and run the `Kipling software <https://labjack.com/blogs/news/kipling-3-1>`__.
+   Screen on the windows monochromator machine when you've connected using Remote Desktop. It will say *Initialized* when ready to use.
 
-If you have any issues with the Moxa devices, go to their address and then:
-
-  - QuickSetup, Select TCP (not Real COM), 9600 Baud, RS-232, Save/Restart
-  - The transmission rate is 9600 Baud, 8 data bits, no parity, 1 stop bit and XON / XOFF flow control
+If you cannot connect to the windows machine at all, you will need to connect it to a monitor, mouse, and keyboard.
```

Comment:
Add that a monitor must be connected via a DisplayPort connector.

Reply:
done